tdvt/tdvt/config_gen/test_config.py

```diff
@@ -250,6 +250,35 @@ def build_config_name(prefix, dsname):
 def build_tds_name(prefix, dsname):
     return prefix + dsname + '.tds'
 
+class TabQueryPath(object):
+    sep = ';'
+
+    def __init__(self, linux_path, mac_path, windows_path):
+        self.linux_path = linux_path
+        self.mac_path = mac_path
+        self.windows_path = windows_path
+        if self.sep in self.linux_path or self.sep in self.mac_path or self.sep in self.windows_path:
+            raise ValueError
+
+    @staticmethod
+    def from_string(paths):
+        split_paths = paths.split(TabQueryPath.sep)
+        if len(split_paths) != 3:
+            raise IndexError
+        t = TabQueryPath(split_paths[0], split_paths[1], split_paths[2])
+        return t
+
+    def to_string(self):
+        return "{0}{1}{2}{3}{4}".format(self.linux_path, self.sep, self.mac_path, self.sep, self.windows_path)
+
+    def get_tabquery_path(self, os):
```

Comment:
Could we rename either this method or the method in `RunTimeTestConfig` to avoid confusion?

Reply:
Updated these.